features/order/managing_orders/seeing_order_shipping_taxes.feature

```diff
@@ -0,0 +1,49 @@
+@managing_orders
+Feature: Seeing taxes of an order
+    In order to know shipping and product taxes
+    As an Administrator
+    I want to be able to see taxes in order
+
+    Background:
+        Given the store operates on a single channel
+        And the store ships to "France" and "Australia"
+        And there is a zone "EU" containing all members of the European Union
+        And there is a zone "The Rest of the World" containing all other countries
+        And default currency is "EUR"
+        And default tax zone is "EU"
+        And the store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
+        And the store has "Low tax" tax rate of 10% for "Clothes" for the rest of the world
+        And the store has "Shipping EU VAT" tax rate of 23% for "Shipping Services" within "EU" zone
+        And the store has "Shipping Low tax" tax rate of 10% for "Shipping Services" for the rest of the world
+        And the store has a product "PHP T-Shirt" priced at "€80.00"
+        And the store has a product "Symfony2 T-Shirt" priced at "€140.00"
+        And it belongs to "Clothes" tax category
+        And the store has "DHL" shipping method with "€10.00" fee within "EU" zone
+        And the store has "DHL-World" shipping method with "€20.00" fee for the rest of the world
+        And shipping method "DHL" belongs to "Shipping Services" tax category
+        And shipping method "DHL-World" belongs to "Shipping Services" tax category
+        And the store allows paying offline
+        And there is a customer "lucy@teamlucifer.com" that placed an order "#00000666"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Seeing taxes of order items and shipping
+        Given the customer bought a single "Symfony2 T-Shirt"
+        And the customer chose "DHL" shipping method to "France" with "Offline" payment
+        When I view the summary of the order "#00000666"
+        Then the order's items total should be "€172.20"
+        And there should be a shipping charge "DHL €10.00"
+        And the order's shipping total should be "€12.30"
+        And the order's tax total should be "€34.50"
+        And the order's total should be "€184.50"
+
+    @ui
+    Scenario: Seeing taxes of order items and shipping in different zone
```

Comment:
I think we don't have to verify that different tax is applied for different zones because that is the responsibility of "tax and zones" scenarios. IMO the above scenario is enough for our case. The background will be much more readable after simplifying that :)